test/util/util.go

```diff
@@ -34,20 +34,20 @@ import (
 
 const interval = time.Millisecond * 250
 
-func NumExpectedJobs(js *jobset.JobSet) int {
-	expectedJobs := 0
+func NumExpectedJobs(js *jobset.JobSet) int32 {
+	var expectedJobs int32 = 0
 	for _, rjob := range js.Spec.ReplicatedJobs {
 		expectedJobs += rjob.Replicas
 	}
 	return expectedJobs
```

Comment:
```suggestion
	return int(expectedJobs)
```
Can we return an int here to avoid the various conversions to int32 when checking number of expected jobs (e.g. `if int32(len(jobList.Items)) != testutil.NumExpectedJobs(js) { ... `)?

Reply:
updated.

Reply:
Doesn't quite work like you said because Replicas is int32.